install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -18,6 +18,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
```

Comment:
Can we add `resourceNames` here such that only access to the `cilium-config` configmap is provided?

Reply:
thanks, added!